### PR TITLE
chore: update event handler location

### PR DESCRIPTION
**Pull request by bradmccoydev**
Update initrepo.yml reference to old event handler file location that is breaking worflow.

Closes issue: https://github.com/keptn-sandbox/keptn-service-template-go/issues/164

Signed-off-by: Brad McCoy <bradmccoydev@gmail.com>

### CODE REVIEW

File: .github/workflows/initrepo.yml [hunk 1]
```diff
@@ -44,7 +44,7 @@ jobs:
           NEW_SERVICE_NAME=$(echo ${{ github.repository }} | sed -e 's/${{ github.repository_owner }}\///')
           ESCAPED_ORIGINAL=$(printf '%s\n' "$ORIGINAL_SERVICE_NAME" | sed -e 's/[\/&]/\\&/g')
           ESCAPED_REPLACE=$(printf '%s\n' "$NEW_SERVICE_NAME" | sed -e 's/[\/&]/\\&/g') 
-          sed -i "s/${ESCAPED_ORIGINAL}/${ESCAPED_REPLACE}/g" .ci_env .gitignore Dockerfile main.go eventhandlers.go eventhandler_test.go test-events/*.json
+          sed -i "s/${ESCAPED_ORIGINAL}/${ESCAPED_REPLACE}/g" .ci_env .gitignore Dockerfile main.go handler/*.go test-events/*.json
```
Comment (TannerGabriel) [inline .github/workflows/initrepo.yml:47]: Can you please update this step's name to include the `action.triggered` eventhandler (in line 42)?

**Review by TannerGabriel — CHANGES_REQUESTED** [review]
Thank you for your contribution. I left a small comment, else LGTM.

File: .github/workflows/initrepo.yml (hunk 1 above)
Comment (bradmccoydev) [reply .github/workflows/initrepo.yml:47]: Done @TannerGabriel

**Review by TannerGabriel — APPROVED** [review]
LGTM

**Pull request merged**